### ISSUE TITLE
Add a delete image feature

### ISSUE THREAD

**Issue by xvolks** [ryouchinsa/Rectlabel-support#16]
Sometimes in the train corpus, we can see the same image twice or an image that we do not want to label (since it has nothing to do here). So we need to delete it.

Now, I 
- open the image folder in Finder, 
- search the image, 
- delete it. 
- On RectLabel, click **Open folder**, 
- choose the same folder, RectLabel reset the current image to 1,
- multiple command + right arrow to search next image (see also #15)
- done

I suggest to just : 
- clic _Edit | Trash current image_ 
- select Yes in confirm box, 
- done.

**Comment by ryouchinsa** [comment]
Thanks for reporting the problem.
We added the short cut key "command + DELETE" to move the image and labels to the deleted folder.
After deleting the image, RectLabel reloads the folder so that you do not have to open the folder again.

![2017-08-10 11 25 07](https://user-images.githubusercontent.com/1954306/29152268-7d73a5aa-7dc1-11e7-88b0-b4241570904c.png)

![2017-08-10 11 26 55](https://user-images.githubusercontent.com/1954306/29152276-894c07a0-7dc1-11e7-8247-d4a1494a34d8.png)

![2017-08-10 11 49 07](https://user-images.githubusercontent.com/1954306/29152355-ff84e216-7dc1-11e7-9bf1-a207f6677e0e.png)

![2017-08-10 11 31 12](https://user-images.githubusercontent.com/1954306/29152277-8b593ebe-7dc1-11e7-902d-1be680a42890.png)

We submitted the new update to Apple.
When the new update is released, we will let you know.
Thank you.
